src/app/App.js

```diff
@@ -77,6 +77,14 @@ class App extends EventEmitter {
 		this.formSelector = 'form[enctype="multipart/form-data"]:not([data-senna-off])';
 
 		/**
+		 * When set to true the route regex parser ignores query string in the path.
+		 * @type {boolean}
+		 * @default false
+		 * @protected
+		 */
+		this.ignoringQueryString = false;
```

Comment:
Let's rename this to: 
`ignoreQueryStringFromRoutePath`
`app.setIgnoreQueryStringFromRoutePath(false);`